### PR TITLE
fix(condo): DOMA-10241 fixes on qa review

### PR DESCRIPTION
**Pull request by toplenboren**

### CODE REVIEW

File: apps/condo/domains/marketplace/utils/clientSchema/Invoice.ts [hunk 1]
```diff
@@ -259,6 +258,9 @@ export function processRowsFromInvoiceTicketForm (rawRows, intl) {
         }
 
         const resultRow = { ...row, toPay: resultToPay }
+        if (!row.measure) {
+            resultRow.measure = undefined
+        }
```
Comment (AleX83Xpert) [inline apps/condo/domains/marketplace/utils/clientSchema/Invoice.ts:263]: You will set `resultRow.measure` to `undefined` even if `row.measure` is 0, false, null, empty string. Is it OK?

File: apps/condo/domains/marketplace/components/Invoice/BaseInvoiceForm.tsx [hunk 2]
```diff
@@ -502,7 +502,7 @@ const ServicesList = ({ organizationId, propertyId, form, currencySymbol, disabl
             const priceObj = get(pricesArray, '0')
             const price = get(priceObj, 'price')
             const isMin = get(priceObj, 'isMin')
-            const measure = get(priceObj, 'measure')
+            const measure = get(priceObj, 'measure', undefined)
```
Comment (AleX83Xpert) [inline apps/condo/domains/marketplace/components/Invoice/BaseInvoiceForm.tsx:505]: Just FYI
if `priceObj.measure` is `null` `get` will return `null`
`undefined` is default value for 3rd parameter, so nothing changed here
Did you meant:
```suggestion
            const measure = get(priceObj, 'measure') || undefined
```
?

Comment (toplenboren) [reply apps/condo/domains/marketplace/components/Invoice/BaseInvoiceForm.tsx:505]: Thanks

**Review by AleX83Xpert — APPROVED** [review]

File: apps/condo/domains/marketplace/utils/clientSchema/Invoice.ts (hunk 1 above)
Comment (AleX83Xpert) [reply apps/condo/domains/marketplace/utils/clientSchema/Invoice.ts:263]: Anyway, strange 3-line logic :)

**Pull request merged**